.travis.yml

```diff
@@ -153,8 +153,8 @@ install:
       travis_retry composer install --no-interaction --ignore-platform-reqs --no-scripts --no-suggest
     elif [[ "$PHPUNIT" == "1" || "$COVERAGE" == "1" || "$PHPLINT" == "1" ]]; then
       # Run composer update as we have dev dependencies locked at PHP ^7.0 versions.
-      travis_retry composer update
-      travis_retry composer install --no-interaction
+      travis_retry composer update --no-interaction --no-scripts
```

Comment:
Could you explain why you made these changes?

Reply:
Found the answer in the Premium PR: "Since we do not want to compile the DI-container (it would fail on variadic constructors)." Will add it to the technical choices.